README.md

```diff
@@ -54,29 +54,34 @@ node build/main.js
 ```
 
 <details>
-  <summary>Server Endpoints/Environment Variables</summary>
+  <summary>Server command line options/endpoints</summary>
 
-**Environment Variables**
-
-- **TOKEN_TTL**: The time in hours for a PO token to be considered valid. While there are no definitive answers on how long a token is valid, it has been observed to be valid for atleast a couple of days. Default: 6
+**Options**
+- `-p, --port <PORT>`: The port on which the server listens.
+- `--verbose`: Use verbose logging
 
 **Endpoints**
 
-- **POST /get_pot**: Accepts a `visitor_data` (unauthenticated), `data_sync_id` (authenticated) or an empty body in the request body. If no identifier is passed, a new unauthenticated `visitor_data` will be generated. Returns `po_token` and the associated identifier `visit_identifier`.
-- **POST /invalidate_caches**: Resets the PO token cache, forcing new tokens to be generated on next fetch.
+- **POST /get_pot**: Generate a new POT.
+  - The request data should be a JSON including:
+    - `visitor_data`: Content binding (optional, a new unauthenticated visitor data will be generated if not passed).
```

Comment:
We need to define what content binding is, and what examples are

Reply:
As of bgutils 3.2, locally generated visitor IDs are no longer supported. We should just make this parameter required